fennel/internal_lib/to_proto/to_proto.py

```diff
@@ -485,13 +485,13 @@ def _extractor_to_proto(
                 inputs.append(feature_to_proto_as_input(f))
         elif isinstance(input, Featureset):
             raise TypeError(
-                f"Extractor input {input} is a Featureset, please use a"
-                f"DataFrame of features"
+                f"{extractor.name} extractor input {input} is a Featureset, please use a"
```

Comment:
Can we use ` to indicate its a user input eg
`{extractor.name}`

Reply:
added